frontend/packages/core/src/Assets/stories/icons.stories.tsx

```diff
@@ -11,13 +11,32 @@ import SlackIcon from "../icons/SlackIcon";
 
 export const AllIcons = () => (
```

Comment:
You can clean this up a little easier so you're not trying to support it all.

```suggestion
export const AllIcons: React.FC<SVGProps> = ({ size }) => (
  <div>
    <FireIcon size={size} />
    <PlusIcon size={size} />
    <ExperimentIcon size={size} />
    <GemIcon size={size} />
    <RocketIcon size={size} />
    <SirenIcon size={size} />
    <SlackIcon size={size} />
  </div>
);

export default {
  title: "Core/Assets/Icons",
  component: AllIcons,
  argTypes: {
    size: {
      options: VARIANTS,
      control: { type: "select" },
    },
  },
} as Meta;
```